### PR TITLE
Translate to german

### PR DESCRIPTION
**Pull request by rrmelcer**
First translate to german

### CODE REVIEW

**Comment by gfreeau** [comment]
Please squash the commits

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+### Addet Functions to this Pomodoro Timer
+- Translate this Applet in German
+- Addet GitHub and BitBucked issues
+- Addet popular Task- and ToDo-Lists
+    - Taskwarrior
+    - ToDoist
+    - Asana
+    - Trello
+---
 # Pomodoro Timer for Cinnamon 2.x, 1.8.x
```
Comment (ncelerier) [inline README.md]: Lines from 1 to 10 are not ok and should be removed. If you want to add popular tasks and to do lists, it must done somewhere else in the document.

Comment (gfreeau) [inline README.md]: Yes, agreed.

**Comment by ncelerier** [comment]
Hello, 

There is something wrong with commit 15302de. I've added a comment in it. As it is currently this can't be merged.

Also, I think only the last 3 commits should be squashed, so that we have: 
- 1 commit for German translations ;
- 1 commit for README.md changes. 

Ideally it could have been 2 differents pull requests.

Also, I think it would be nice if you could fix spelling mistakes in the commit messages (tamplate -> template, addet -> added).

**Pull request closed without merging**